server_cmds_test.go

```diff
@@ -75,15 +75,19 @@ func TestCmdsServer(t *testing.T) {
 			return
 		}
 		expected := &Server{
-			Status:                                 "template",
+			Status:                                 "online",
 			MaxClients:                             32,
 			Name:                                   "Test Server",
+			Uptime:                                 6802178,
+			Port:                                   9987,
+			AutoStart:                              true,
 			AntiFloodPointsNeededCommandBlock:      150,
 			AntiFloodPointsNeededIPBlock:           250,
 			AntiFloodPointsTickReduce:              5,
 			ComplainAutoBanCount:                   5,
 			ComplainAutoBanTime:                    1200,
 			ComplainRemoveTime:                     3600,
+			Created:                                time.Unix(2, 0),
```

Comment:
This looks wrong, I very much doubt it was created in 1970's

Reply:
aren't those values just for testing purposes only? I only increased the value to 2 to mitigate some weird boolean / integer mixup.

Reply:
The values where real values from a real request, then confirmed to be the correct conversion where needed.

This helps avoid broken made up data.